src/LaravelModulesServiceProvider.php

```diff
@@ -12,6 +13,10 @@ class LaravelModulesServiceProvider extends ModulesServiceProvider
      */
     public function boot()
     {
+        if (!Schema::hasTable(config('modules.activators.database.table'))) {
```

Comment:
This can't be here.

Reply:
It has to be since you will get an error that the database does not have a table 'modules' because the provider is booted before you can migrate.

Reply:
This is the reason all tests fail. This is also related to the implementation in a place where implementation can't be referenced.